lib/chef-acceptance/suite_runner.rb

```diff
@@ -0,0 +1,94 @@
+require 'chef-acceptance/output_formatter'
+require 'chef-acceptance/chef_runner'
+
+module ChefAcceptance
+  # Responsible for running one or more suites and displaying statistics about
+  # the runs to the user
+  class SuiteRunner
+    attr_reader :force_destroy
+
+    def initialize(force_destroy = false)
```

Comment:
I would make the constructor take an `options` hash so that when we need more things in the future we will not need to change the signature of it.